rpt/OH007_lab_monthly_report.jrxml

```diff
@@ -22,23 +22,23 @@
    select EXA_DESC AS EXAM, LAB_MATERIAL AS MATERIAL, 'Tests done' AS CATEGORY, count(*) AS COUNT 
    from OH_LABORATORY, OH_EXAM
    where LAB_EXA_ID_A = EXA_ID_A
-   and month(LAB_EXAM_DATE) = $P{month} and year(LAB_EXAM_DATE) = $P{year}
+   and month(LAB_CREATED_DATE) = $P{month} and year(LAB_CREATED_DATE) = $P{year}
```

Comment:
the new field to rely on is `LAB_DATE`, `LAB_CREATED_DATE` is for audit purpose only

Reply:
same for all reports

Reply:
Change made in all reports

Reply:
Thanks!